src/main/java/org/opentripplanner/middleware/controllers/api/MonitoredTripController.java

```diff
@@ -19,6 +20,7 @@
 
 import static com.beerboy.ss.descriptor.MethodDescriptor.path;
 import static com.mongodb.client.model.Filters.eq;
+import static org.opentripplanner.middleware.tripmonitor.jobs.MonitorAllTripsJob.monitoredTripLocks;
```

Comment:
Might be nice to provide an API for this (with methods like `lockTrip` and `isTripLocked`) instead of importing the HashMap.

Reply:
Refactored in 6948fab3bf5ead8b87a4b47676a38fe9a6e1691f